### ISSUE TITLE
Incompatibility with Ember-CLI 3.5.0 & Broccoli 2 

### ISSUE THREAD

**Issue by nikdojo** [sir-dunxalot/ember-cli-concat#45]
Hi!

There is an issue building with Ember CLI 3.5 that uses Broccoli v2 and `ember-cli-concat`:
```
WARNING: Invalid Broccoli2 node detected, falling back to broccoli-builder. Broccoli error:
---------------
Remover: The .read/.rebuild API is no longer supported as of Broccoli 1.0. Plugins must now derive from broccoli-plugin. https://github.com/broccolijs/broccoli/blob/master/docs/broccoli-1-0-plugin-api.md
used as input node to BroccoliMergeTrees
-~- created here: -~-
    at BroccoliMergeTrees.Plugin (/node_modules/broccoli-plugin/index.js:7:31)
    at new BroccoliMergeTrees (/node_modules/broccoli-merge-trees/index.js:42:10)
    at BroccoliMergeTrees (/node_modules/broccoli-merge-trees/index.js:36:53)
    at Class.postprocessTree (/node_modules/ember-cli-concat/index.js:324:21)
-~- (end) -~----------------
```

Could you please update the addon to fix it? Thx.
Cheers!

**Comment by rwwagner90** [comment]
Hi @nikdojo, we had implemented support recently, but never published a new version. I just published 2.0.1, please give it a try!

**Comment by nikdojo** [comment]
Aaaaaaawesome!
Thx man!
